include/switcher.php

```diff
@@ -10,6 +10,16 @@
  */
 class PLL_Switcher {
 
+	public static function create( $polylang ) {
+		if ( $polylang instanceof PLL_Frontend ) {
+			return new PLL_Frontend_Switcher();
+		} else if ( $polylang instanceof  PLL_Admin ) {
+			return new PLL_Admin_Switcher();
+		} else {
+			return '';
```

Comment:
A string is not a return value that I would expect here.

I would suggest we throw an exception at this point, as this case could only happen if we made an error during development.